docs/extend/legacy_plugins.md

```diff
@@ -15,8 +15,6 @@ keywords: "Examples, Usage, plugins, docker, documentation, user guide"
      will be rejected.
 -->
 
-# Use Docker Engine plugins
```

Comment:
That's probably better for a follow-up where we remove Engine from titles in all relevant places, yes?